kafkaConsumer/src/main/java/com/consumerservice/model/output/OrderCreatedModel.java

```diff
@@ -0,0 +1,19 @@
+package com.consumerservice.model.output;
+
+import com.consumerservice.model.input.OrderInputModel;
+import lombok.Data;
+
+import java.util.Date;
+
+@Data
+public class OrderCreatedModel {
+    private final String eventName;
+    private Date orderEntityCreationDate;
+    private OrderInputModel orderData;
+
+    public OrderCreatedModel(Date orderEntityCreationDate, OrderInputModel orderData) {
+        this.eventName = "orders/created";
```

Comment:
i would extract out to a const or Enum instead something like:
```suggestion
        this.eventName =  EventType.CREATED;
```